tests/system/test_client.py

```diff
@@ -1800,11 +1790,11 @@ def test_dbapi_fetchall_from_script(self):
         row_tuples = [r.values() for r in rows]
         self.assertEqual(row_tuples, [(5, "foo"), (6, "bar"), (7, "baz")])
 
-    @unittest.skipIf(
-        bigquery_storage is None, "Requires `google-cloud-bigquery-storage`"
-    )
-    @unittest.skipIf(pyarrow is None, "Requires `pyarrow`")
     def test_dbapi_fetch_w_bqstorage_client_large_result_set(self):
+        pytest.importorskip("google-cloud-bigquery-storage")
```

Comment:
importorskip needs to be the import name not the package name

```suggestion
        bigquery_storage = pytest.importorskip("google.cloud.bigquery_storage")
```

https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-importorskip